packages/openneuro-server/src/models/badAnnexObject.js

```diff
@@ -0,0 +1,18 @@
+import mongoose from 'mongoose'
+
+const badAnnexObject = new mongoose.Schema(
+  {
+    datasetId: String, // OpenNeuro id
+    snapshot: String,
+    annexKey: String,
+    removed: { type: Boolean, default: false },
+    remover: { _id: String },
+    flagged: { type: Boolean, default: false },
+    flagger: { _id: String },
+  },
+  { timestamps: { createdAt: 'created_at' } },
+)
+
+const BadAnnexObject = mongoose.model('BadAnnexObject', badAnnexObject)
+
+export default BadAnnexObject
```

Comment:
Before merging this, the new collection should have a TypeScript interface to avoid conflicts with #2054

This model could use an index on datasetId, annexKey as well.

Reply:
@david-nishi please add a tooltip explaining what this does.